source/docs/software/pathplanning/robot-characterization/analyzing-feedback.rst

```diff
@@ -7,12 +7,12 @@ Feedback Analysis
 
 .. warning:: The feedback gain calculation assumes that there is no mechanical backlash, sensor noise, or phase lag in the sensor measurement.  While these are reasonable assumptions in many situations, none of them are strictly true in practice.  In particular, many "smart motor controllers" (such as the ``Talon SRX``, ``Talon FX``, and ``SPARK MAX``) have default settings that apply substantial :ref:`low-pass filtering <docs/software/advanced-controls/filters/introduction:Introduction to Filters>` to their encoder velocity measurements, which introduces a significant amount of phase lag.  This can cause the calculated gains for velocity loops to be unstable.  To rectify this, either decrease the amount of filtering through the controller's API, or reduce the magnitude of the PID gains - it has been found that shrinking gains by about a factor of 10 works well for most default filtering settings.
```

Comment:
We should suggest using the position and velocity measurement delay boxes first. It's the more mathematically optimal way to account for the latency involved (the controller gain is as aggressive as it can be in the presence of delay). We can still mention the 10x scaling rule of thumb afterward.

We don't mention the measurement delay boxes at all, so idk if here or further down is better.

Reply:
Also, this is a lot of content for a admonition. I'd make this a standard paragraph.

Reply:
Do we even need this paragraph, since the motor controller presets should account for at least the phase lag?

Reply:
Just waiting on this to get resolved. @prateekma @calcmogul comments?

Reply:
It's not yet resolved.

Reply:
Once that issue is resolved I'll fixup this and update the image for it as well.